packages/python/plotly/plotly/io/_base_renderers.py

```diff
@@ -861,7 +861,7 @@ def render(self, fig_dict):
         filename_html = filename_root + ".html"
         filename_png = filename_root + ".png"
         figure = return_figure_from_figure_or_data(fig_dict, True)
-        _ = write_html(fig_dict, file=filename_html)
+        _ = write_html(fig_dict, file=filename_html, include_plotlyjs=False)
```

Comment:
hmm so how does Plotly.js get included?

Reply:
the first time I mean

Reply:
Hum good question. I had checked that with `include_plotlyjs=False` the plotly cdn was included in the html files (which is weird, but it did include the cdn). I switched to `include_plotlyjs='cdn'` instead.